frontend/packages/console-dynamic-plugin-sdk/scripts/package-definitions.ts

```diff
@@ -169,7 +179,7 @@ export const getWebpackPackage: GetPackageDefinition = (
       ...parseDepsAs(rootPackage, { 'lodash-es': 'lodash' }, missingDepCallback),
     },
     peerDependencies: {
-      typescript: '>=4.5.5',
+      typescript: '>=' + getMinDepVersion(rootPackage, 'typescript', missingDepCallback),
```

Comment:
Note :memo: this ensures that `typescript` peer dependency in `packages/console-dynamic-plugin-sdk/dist/webpack/package.json` reflects the actual TS version used when building Console plugin SDK packages.